README.md

```diff
@@ -3,152 +3,237 @@ Table of Contents
     * [Monolithic With Compile Masters ( Running on the MoM )](#monolithic-with-compile-masters--running-on-the-mom-)
+      * [Hiera data Example](#hiera-data-example-2)
+      * [Class Definition Example](#class-definition-example-1)
     * [Split With Compile Masters ( Running on the MoM )](#split-with-compile-masters--running-on-the-mom-)
+      * [Hiera data Example](#hiera-data-example-3)
+      * [Class Definition Example](#class-definition-example-2)
     * [Running on PE 3\.8](#running-on-pe-38)
-    * [Other Option](#other-option)
+    * [Temporary Install](#temporary-install)
+    * [Alternate Option for Multi\-node Metrics Collection](#alternate-option-for-multi-node-metrics-collection)
   * [What do you get](#what-do-you-get)
     * [Grepping for Metrics](#grepping-for-metrics)
       * [Puppetserver](#puppetserver)
       * [PuppetDB](#puppetdb)
 
 # How to use
 
-```
-include pe_metric_curl_cron_jobs
-```
+Install the module with `puppet module install npwalker-pe_metric_curl_cron_jobs` or add it to your Puppetfile.
+
+To start data collection you will need to classify your puppet master with the `pe_metric_curl_cron_jobs` class using your preferred classification method.
 
-If you do not want to manage this long term and want to get it up and running quickly you can run it via puppet apply.
+The following examples show how to configure the parameters to work in different setups but we assume you will always classify on the node that is the CA master.  The preferred method is to `include` the module and then provide hiera
+data for the parameters.
 
 ## Monolithic Install
 
-```
-cd /tmp;
-git clone https://github.com/npwalker/pe_metric_curl_cron_jobs;
-puppet apply -e "class { 'pe_metric_curl_cron_jobs': }" --modulepath .
-```
+### Hiera data Example
+
+None needed for a monolithic install
+
+### Class Definition
+
+~~~
+include pe_metric_curl_cron_jobs
+~~~
 
 ## Split Install ( Running on the Master )
 
-```
-cd /tmp;
-git clone https://github.com/npwalker/pe_metric_curl_cron_jobs;
-puppet apply -e "class { 'pe_metric_curl_cron_jobs' : puppetdb_hosts => ['split-puppetdb.domain.com'] }" --modulepath .
-```
+### Hiera data Example
+
+~~~
+pe_metric_curl_cron_jobs::puppetdb_hosts:
+ - 'split-puppetdb.domain.com'
+~~~
+
+### Class Definition Example
+
+~~~
+class { 'pe_metric_curl_cron_jobs':
+  puppetdb_hosts => ['split-puppetdb.domain.com']
+}
+~~~
 
 ## Monolithic With Compile Masters ( Running on the MoM )
 
-```
-cd /tmp;
-git clone https://github.com/npwalker/pe_metric_curl_cron_jobs;
-puppet apply -e "class { 'pe_metric_curl_cron_jobs' : puppet_server_hosts => ['compile-master-1.domain.com', 'compile-master-2.domain.com'] }" --modulepath .
-```
+### Hiera data Example
+
+~~~
+pe_metric_curl_cron_jobs::puppetserver_hosts:
+ - 'master-1.domain.com'
+ - 'compile-master-1.domain.com'
+ - 'compile-master-2.domain.com'
+~~~
+
+### Class Definition Example
+
+~~~
+class { 'pe_metric_curl_cron_jobs':
+  puppet_server_hosts => [
```

Comment:
this should be `puppetserver` (current API) instead of `puppet_server` (deprecated API)